gen/definitions/bgp_address_family.yaml

```diff
@@ -46,24 +46,144 @@ attributes:
       - ipv6-mvpn
       - l2vpn-vpls
       - ipv4-mdt
-    default_value: 'ipv4-ucast'
-    example: 'ipv4-ucast'
+    default_value: "ipv4-ucast"
+    example: "ipv4-ucast"
   - nxos_name: critNhTimeout
     tf_name: critical_nexthop_timeout
-    type: Int64
-    description: 'The next-hop address tracking delay timer for critical next-hop reachability routes.'
-    min_int: 1
-    max_int: 4294967295
-    default_value: 3000
-    example: 1800
+    type: String
+    description: "The next-hop address tracking delay timer for critical next-hop reachability routes."
+    default_value: "crit"
+    example: 2500
   - nxos_name: nonCritNhTimeout
     tf_name: non_critical_nexthop_timeout
+    type: String
+    description: "The next-hop address tracking delay timer for non-critical next-hop reachability routes."
+    default_value: "noncrit"
+    example: 8000
+  - nxos_name: advertL2vpnEvpn
+    tf_name: adv_l2vpn_evpn
```

Comment:
For the `tf_name` we try to avoid abbreviations and expand names where it makes sense. So I would suggest `advertise_l2vpn_evpn`.